app/react-native/package.json

```diff
@@ -72,6 +72,6 @@
     "babel-core": "^6.26.0 || ^7.0.0-0",
     "babel-runtime": ">=6.0.0",
     "react": "*",
-    "react-native": "0.52.2"
+    "react-native": ">=0.51.0"
```

Comment:
rn is actually pretty tricky to upgrade. it needs matched versions of all the tooling as well for everything to bundle, compile, and work. I still think this is the right move though.